packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -372,3 +372,201 @@ function sendDtmfTones() {
+  selectElm.innerHTML = 'Select';
+
+  removeElm.onclick = () => webex.devicemanager.remove(device.id);
+  removeElm.type = 'button';
+  removeElm.value = device.id;
+  removeElm.innerHTML = 'webex.devicemanager.remove()';
+
+  detailsElm.innerHTML = `${device.deviceInfo.description} [${device.id}]`;
+
+  itemElm.appendChild(selectElm);
+  itemElm.appendChild(removeElm);
+  itemElm.appendChild(detailsElm);
+
+  return itemElm;
+}
+
+function generateMeetingsDevicesSearchItem(device) {
+  const itemElm = document.createElement('div');
+  const selectElm = document.createElement('button');
+  const detailsElm = document.createElement('label');
+
+  console.log('device', device);
+
+  itemElm.setAttribute('id', `meeting-list-item-${device.id}`);
+  itemElm.setAttribute('key', device.id);
+
+  selectElm.onclick = () => selectDevice(device);
+  selectElm.setAttribute('type', 'button');
+  selectElm.setAttribute('value', device.id);
+  selectElm.innerHTML = 'Select';
+
+  detailsElm.innerHTML = `${device.description || device.name} [${device.id}]`;
+
+  itemElm.appendChild(selectElm);
+  itemElm.appendChild(detailsElm);
+
+  return itemElm;
+}
+
+function deviceGetAudioState() {
+  console.log('MeetingsDevices#refreshDevicesList()');
+
+  webex.devicemanager.getAudioState()
+    .then((state) => {
+      currentDeviceAudioStateElm.innerHTML = JSON.stringify(state, null, 2);
+    });
+}
+
+function refreshDevicesList() {
+  console.log('MeetingsDevices#refreshDevicesList()');
+
+  devicesListItemsElm.innerHTML = '';
+
+  webex.devicemanager.refresh()
+    .then((devices) => {
+      if (devices.length === 0) {
+        devicesListItemsElm.innerHTML =
+          'There are currently no meetings devices to display';
+
+        return;
+      }
+
+      devices.forEach(
+        (device) => devicesListItemsElm.appendChild(
+          generateMeetingsDevicesListItem(device)
+        )
+      );
+    });
+}
+
+function searchForDevices() {
+  console.log('DevicesControls#searchForDevices()');
+  findDevicesStatusElm.innerHTML = 'Searching...';
+  findDevicesListElm.innerHTML = '';
+
+  const searchQuery = findDevicesQueryElm.value;
+
+  if (searchQuery && searchQuery.length < 3) {
+    const msg = 'device query must contain 3 characters or more';
+
+    findDevicesStatusElm.innerHTML = msg;
+
+    throw new Error(msg);
+  }
+
+  webex.devicemanager.search({searchQuery})
+    .then((results) => {
+      results.forEach(
+        (device) => findDevicesListElm.appendChild(
+          generateMeetingsDevicesSearchItem(device)
+        )
+      );
+
+      findDevicesStatusElm.innerHTML = '';
+    })
+    .catch((error) => {
+      findDevicesQueryElm.innerHTML = error.message;
+    });
+}
+
+function claimPersonalMeetingRoom() {
+  console.log('DevicesControls#claimPersonalMeetingRoom()');
+
+  pmrDetailsElm.innerHTML = 'Attempting to claim';
+
+  const pmrId = pmrIdElm.value;
+  const pmrPin = pmrPinElm.value;
+
+  if (!pmrId && !pmrPin) {
```

Comment:
Looking at the error thrown below, did you mean `!pmrId || !pmrPin`? It seems as if you are requiring both to be present, unless I misunderstood.

Reply:
Good catch! updating this now.